### PR TITLE
[hab3_merge] Main different policies per agent

### PR DESCRIPTION
**Pull request by xavierpuigf**
## Motivation and Context

This is needed so that we can define different policy types for each agent. It may be useful to be able to train a multi-agent task where one agent is run by a planner and the other one is run by a neural net, or one has a low level policy and the other has a high level policy.

Cherry-picked from this PR: https://github.com/facebookresearch/habitat-lab/commit/26a9696d0fd8aad795787ad49d19b89e3359db61

## How Has This Been Tested

Passes Test + End to End Testing.

## Types of changes

- **\[Development\]** A pull request that add new features to the [habitat-lab](/habitat-lab) task and environment codebase. Development Pull Requests must be small (less that 500 lines of code change), have unit testing, very extensive documentation and examples. These are typically new tasks, environments, sensors, etc... The review process for these Pull Request is longer because these changes will be maintained by our core team of developers, so make sure your changes are easy to understand!

## Checklist

- [X] My code follows the code style of this project.
- [X] I have updated the documentation if required.
- [X] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [X] I have completed my CLA (see **CONTRIBUTING**)
- [X] I have added tests to cover my changes if required.

### CODE REVIEW

File: habitat-baselines/habitat_baselines/config/default_structured_configs.py [hunk 1]
```diff
@@ -104,7 +104,6 @@ class CenterCropperConfig(ObsTransformConfig):
 
 
 cs.store(
-    package="habitat_baselines.rl.policy.obs_transforms.center_cropper",
```
Comment (rpartsey) [inline habitat-baselines/habitat_baselines/config/default_structured_configs.py:107]: Why did you remove all of the packages here?

Comment (xavierpuigf) [reply habitat-baselines/habitat_baselines/config/default_structured_configs.py:107]: It is needed because now obs_transforms need to be accessed as:

habitat_baselines.rl.policy.**agent_name**.obs_transforms.center_cropper
